app/assets/stylesheets/application.scss

```diff
@@ -66,6 +66,14 @@ html {
 }
 
 .toast {
-  &.success { background-color: $success-color; }
-  &.error { background-color: $error-color; }
+  background-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`